### DISCUSSION TITLE
How to set up flexric and gNB(with open5gs) on two host machine

### DISCUSSION THREAD

**Discussion by peihaoY** [srsran/srsRAN_Project#456] [General Help] [answered]
I have follow this link to run near-real time RIC and gNB, https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html, it works. Now I am trying to set flexric and gNB(with open5gs) on different host machines. I changed the ric.conf and n310.yml, but "failed to connect SCTP to 35.12.208.66:36421, error="Network is unreachable". Could you help me to have a look at my config files?
 ric.conf and n310.yml:
[ric and ngb config.zip](https://github.com/srsran/srsRAN_Project/files/14185488/ric.and.ngb.config.zip)
I find the RIC running on a different machine
![image](https://github.com/srsran/srsRAN_Project/assets/147773058/7d352631-b36d-497c-8511-fdaa9b1d6a0a)
Could you tell me how to change the two IP address?
addr: 127.0.0.1                   # RIC IP address
bind_addr: 127.0.0.100           # A local IP that the E2 agent binds to for traffic from the RIC. ONLY required if running the RIC on a separate machine.

**Comment by pgawlowicz** [comment, marked as answer]
you need to add `bind_addr` under e2 section in gnb.config and set it to the IP address of the interface that connects to RIC.

**Reply by pgawlowicz** [reply]
@peihaoY any updates on your issue?

**Reply by peihaoY** [reply]
Now it works fine. I can control it with the wifi IPV4 address or I can control it by connecting to the router. Thanks a lot!

**Comment by yagoda** [comment]
Hi @peihaoY,
In preparation for our upcoming workshop in DC, we are contacting some users of our E2 solution. Can you please share a few lines about your experience with the E2 interface, what you were trying to achieve, and how it worked out for you?

Kind Regards,
Justin
